test/spec/label.js

```diff
@@ -272,9 +272,9 @@ describe("label", function () {
     let evaluatedLabelAndGloss = (locales, properties) => {
       let evaluated = evaluatedExpression(locales, properties);
       if (typeof evaluated === "string") {
-        return [evaluated, null];
+        return [evaluated];
       }
-      return [evaluated.sections[0].text, evaluated.sections[4].text];
+      return [evaluated.sections[0].text, evaluated.sections[4]?.text];
```

Comment:
As a result of the manual cast above, this expression no longer returns a literal string, always a `format` object.